packages/airnode-validator/src/ois/ois.ts

```diff
@@ -22,20 +21,17 @@ export const operationParameterSchema = z.object({
 
 export const fixedParameterSchema = z.object({
   operationParameter: operationParameterSchema,
-  // TODO: This could be any JSON value
   value: z.string(),
 });
 
 export const methodSchema = z.union([z.literal('get'), z.literal('post')]);
 
 export const endpointOperationSchema = z.object({
   method: methodSchema,
-  // TODO: Validate URL path
```

Comment:
I have noted this down and will groom this this to an existing issue or create new.